.gitignore

```
@@ -0,0 +1,8 @@
+
+/.vs
+/backend/.vs/CarRentalSystem
+/backend/.vs
+/frontend/node_modules
+/backend/CustomerAPI/obj/Debug/net8.0
+/backend/CustomerAPI/obj/Debug/net8.0
+/backend/CustomerAPI/obj/Debug
```

```suggestion
/bin/
```
No need to checkin the bin folder as it gets generated on build